onnxruntime/core/providers/rocm/rocm_execution_provider.cc

```diff
@@ -1208,841 +1209,841 @@ KernelCreateInfo BuildKernelCreateInfo<void>() {
 
 static Status RegisterRocmKernels(KernelRegistry& kernel_registry) {
   static const BuildKernelCreateInfoFn function_table[] = {
-    BuildKernelCreateInfo<void>,  //default entry to avoid the list become empty after ops-reducing
```

Comment:
lots of formatting changes in this file, can ignore whitespace when viewing diff